DTS-STN/Service-Canada-Labs
# Signup pages 404

# Description

This change causes the signup pages to 404 in production.

## Acceptance Criteria

Get the acceptance criteria for the story and/or task you are working on and put it all here.

## Test Instructions

1. set ENVIRONMENT=production in your .env
2. yarn build and yarn start
3. navigate to http://localhost:3000/en/signup
4. navigate to http://localhost:3000/en/signup-info
5. navigate to http://localhost:3000/en/signup-review
6. verify the above links all lead to the 404 page

## Checklist
